lab/Dockerfile

```diff
@@ -42,7 +44,8 @@ RUN umask 0 && \
   mamba install -c ${CHANNEL} -c in-core/label/rc -y \
      pyincore=${PYINCORE} \
      pyincore-viz=${PYINCORE_VIZ} \
-     pyincore-data=${PYINCORE_DATA} && \
+     pyincore-data=${PYINCORE_DATA} \
+    pyincore-incubator=${PYINCORE_INCUBATOR} && \
```

Comment:
This indentation shouldn't matter?

Reply:
oh somehow it worked without error, so it works not fixing it.